src/lib/evil/unposix/sys/time.h

```diff
@@ -1,26 +1,43 @@
-#ifndef TIME_H
-#define TIME_H
-
-#include "unimplemented.h"
+#ifndef SYS_TIME_H
+#define SYS_TIME_H
 
+#ifndef WIN32_LEAN_AND_MEAN
+# define WIN32_LEAN_AND_MEAN 1
```

Comment:
what is this ?

Reply:
Window.h automatically includes a lot of other headers, using WIN32_LEAN_AND_MEAN it only includes what is explicitly included. I think that, for safety, everywhere you include windows.h you should verify if WIN32_LEAN_AND_MEAN is set (Am I right @JPTIZ ? ).

Reply:
> Window.h automatically includes a lot of other headers, using WIN32_LEAN_AND_MEAN it only includes what is explicitly included. I think that, for safety, everywhere you include windows.h you should verify if WIN32_LEAN_AND_MEAN is set (Am I right @JPTIZ ? ).

Mostly that. Not exactly only what is explicitly included, just prevents messing up trying to use two conflicting APIs at the same time (e.g. winsock and winsock2).